kyber_group.go

```diff
@@ -111,6 +111,13 @@ func (s *Suite) GT() kyber.Group {
 	return NewGroupGT()
 }
 
+func (s *Suite) ValidatePairing(p1, p2, p3, p4 kyber.Point) bool {
```

Comment:
comment / expected properties of p1-p4 would be nice :)
or like "this is an optimization of Validate(Pair(p1,p2), Pair(p3,p4))"